website/blog/bazels-remote-caching-and-remote-execution-explained.md

```diff
@@ -254,7 +254,7 @@ it can result in slower builds.
 
 ### Flags
 
-<a id="remote_executor-flag"></a>
+#### `remote_executor-flag`
```

Comment:
nit: I think it would look a little nicer to have the headings look like `--remote_executor` instead of `remote_executor-flag`. You could override the generated ID using the `{#ID}` syntax

```
#### `--remote_executor` {#remote_executor-flag}
```

Reply:
Sounds great!

Reply:
```suggestion
#### `--remote_executor` {#remote_executor-flag}
```